src/rtk/features/delegableApi/delegableApiSlice.ts

```diff
@@ -181,30 +185,40 @@ const delegableApiSlice = createSlice({
       } else {
         state.presentedApiList = delegableApiSearchPool;
```

Comment:
I'm getting error because scopes aren't a nullable property in DelegableApi and you're pushing an object without scopes to an DelegableApi object. Should the presentedApiList still contain scopes after it has been filtered?

Reply:
presentedApiList should absolutely contain scope after filtering. Nicely spotted! 😊